package.json

```diff
@@ -18,7 +18,8 @@
   },
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "dependencies": {
-    "classnames": "^2.2.5"
+    "classnames": "^2.2.5",
+    "react-bootstrap": "^0.30.8"
   },
```

Comment:
IIUC There won't be any errors in case when application will use different version of react-bootstrap as a dependency. (see last example at https://snyk.io/blog/whats-an-npm-dependency/).

peerDependency is IMHO not a good choice because it forces the application to include it as a dependency regardless of the fact that it directly uses it. patternfly or react is a peerDependency because if the application uses patternfly-react it is expected that it is a react application which uses patternfly styles. Although it does not need to use react-bootstrap.

To make react-bootstrap an optionalDependency, then we'd have to make sure that we handle cases when react-bootstrap is not available in patternfly-react codebase (https://docs.npmjs.com/files/package.json#optionaldependencies) I don't think we want to go that way either, or do we?